float32[6] coxa_angle_deg
float32[6] femur_angle_deg
float32[6] tibia_angle_deg
